docs/how-to-guides/themes/theme-support.md

```diff
@@ -315,7 +315,7 @@ Themes can opt out of generated block layout styles that provide default structu
 add_theme_support( 'disable-layout-styles' );
 ```
 
-For themes looking to customize `blockGap` styles or block spacing, see [the developer docs on Global Settings & Styles](/docs/how-to-guides/themes/theme-json/#what-is-blockgap-and-how-can-i-use-it).
+For themes looking to customize `blockGap` styles or block spacing, see [the developer docs on Global Settings & Styles](/docs/how-to-guides/themes/global-settings-and-styles/#what-is-blockgap-and-how-can-i-use-it).
```

Comment:
```suggestion
For themes looking to customize `blockGap` styles or block spacing, see [the developer docs on Global Settings & Styles](/docs/how-to-guides/themes/global-settings-and-styles.md#what-is-blockgap-and-how-can-i-use-it).
```

If the URL is a relative path starting with `/docs` and has an anchor, I think the link will not work properly on GitHub unless you include the markdown file extension.

https://github.com/WordPress/gutenberg/assets/54422211/79f291e8-5e72-40cd-9c97-8ab94b14f9b2

Reply:
Ahh, good catch.